<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::CloseMethodDialog</class>
 <widget class="QDialog" name="ns::CloseMethodDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>229</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About YesPo Service Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

/* css of caption */
#frameCaption {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #0078c7, stop: 1.0 #016aae);
	border-left: 1px solid #57A3D3;
	border-top: 1px solid #57A3D3;
	border-right: 1px solid #57A3D3;
	border-bottom: 1px solid #0b619c;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
}

#frameCaption #labelCaptionLogo {
	background-image: url(:/images/ui/logo.png);
	background-repeat: none;
	background-position: center;
	max-width: 32px;
	min-width: 32px;
	max-height: 32px;
	min-height: 32px;
}

#frameCaption #labelTitle {
	font-size: 13px;
	font-weight: bold;
	color: #ffffff;
}

#frameCaption #frameSystemMenu {
	border: 0px;
}

#frameCaption #frameSystemMenu #sysPanelContainer {
	border: 0px;
}

#frameCaption #pushButtonClose {
	border-image: url(:/images/ui/caption_close.png);
}

#frameCaption #pushButtonClose:hover {
	border-image: url(:/images/ui/caption_close_hover.png);
}

#frameCaption #pushButtonClose:pressed {
	border-image: url(:/images/ui/caption_close_pressed.png);
}

#frameTop {
	border: 1px solid #BDE0F4;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #9BD0F0, stop: 0.2 #EFF8FF);
}

/* end of css caption */

#widgetBottom {
	background: #E1F1FE;
	border: 1px solid #BDE0F4;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

#pushButton {
	color: #FFFFFF;
	border-image: url(:/images/ui/splash_answer.png) 0 3 0 3;
	border-top: 0px transparent;
	border-bottom: 0px transparent;
	border-right: 3px transparent;
	border-left: 3px transparent;
}

#pushButton:hover {
	border-image: url(:/images/ui/splash_answer_hover.png) 0 3 0 3;
}

#pushButton:pressed {
	border-image: url(:/images/ui/splash_answer_pressed.png) 0 3 0 3;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameCaption">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCaptionLogo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>YesPo Service Client</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="sysPanelContainer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="sysPanel" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameTop">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widgetGroup" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButtonTray">
           <property name="text">
            <string>Show tray icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonQuit">
           <property name="text">
            <string>Quit application</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetBottom" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxRemember">
        <property name="text">
         <string>Do not show this again</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>77</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>22</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ns::CloseMethodDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ns::CloseMethodDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
